components/prism-http.js

```diff
@@ -1,21 +1,44 @@
 (function (Prism) {
 	Prism.languages.http = {
 		'request-line': {
-			pattern: /^(?:POST|GET|PUT|DELETE|OPTIONS|PATCH|TRACE|CONNECT)\s(?:https?:\/\/|\/)\S*\sHTTP\/[0-9.]+/m,
+			pattern: /^(?:GET|HEAD|POST|PUT|DELETE|CONNECT|OPTIONS|TRACE|PATCH|PRI|SEARCH)\s(?:https?:\/\/|\/)\S*\sHTTP\/[0-9.]+/m,
```

Comment:
1. Please add the new methods to [`request-line` test](https://github.com/PrismJS/prism/blob/master/tests/languages/http/request-line_feature.test).
2. Please use _one_ pattern to match the HTTP version. Right now you use `HTTP\/[0-9.]+` here and `HTTP\/1.[01]` in `response-line`.